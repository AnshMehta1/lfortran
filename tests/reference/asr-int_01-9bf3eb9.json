{
    "basename": "asr-int_01-9bf3eb9",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/int_01.f90",
    "infile_hash": "77368d4c89de62e3f870622426296e4ddb2cf8cdd43b42822433401a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-int_01-9bf3eb9.stderr",
    "stderr_hash": "8341fb653e48fb782a653dbdb51a84f43e57eaaac0f26d8fa911eaa8",
    "returncode": 2
}