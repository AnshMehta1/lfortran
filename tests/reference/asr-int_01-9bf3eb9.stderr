semantic error: kind argument to int(a, kind) is not a constant integer
 --> tests/errors/int_01.f90:2:36
  |
2 |     real(8), parameter :: ar1(3) = int([1, 2, 3], [8, 8, 8])
  |                                    ^^^^^^^^^^^^^^^^^^^^^^^^^ 
